tools/integration_tests/util/setup/setup.go

```diff
@@ -236,6 +249,15 @@ func UnMountAndThrowErrorInFailure(flags []string, successCode int) {
 	if successCode != 0 {
 		f := strings.Join(flags, " ")
 		log.Print("Test Fails on " + f)
+
+		// Logfile name will be gcsfuse-failed-integration-test-log-xxxxx
+		failedlogsFileName := "gcsfuse-failed-integration-test-logs-" + GenerateRandomString(5)
```

Comment:
why do you need such a big name. I am assumig the log is different for every package. how will you which log corresponds which failure?

Reply:
We are printing log file name after test failure. If test will fail one can find that file name in artifacts.
logfile is different for every mount.

Reply:
Got it. Also why do you have this logic of generating random string ? why can't you directly copy the log file with its original name into some destination folder and use that as regex_match for artifacts.

Reply:
Logfile name is gcsfuse.log only, it is in the directory /tmp/gcsfuse_readwrite_test_xxxxxxxx/gcsfuse.log. You mean to copy whole path? If yes, than we have to first create directory /tmp/gcsfuse_readwrite_test_xxxxxxxx/ in kokoro artifacts. 

I think what I did previously by creating file name with random string seems okay to me. Let me know what correction is required. I'll work accordingly.

Reply:
Okay. 
Atleast remove gcsfuse from this: gcsfuse-failed-integration-test-logs-

name looks too big.